README.md

```diff
@@ -154,6 +155,75 @@ export default function () {
 }
 ```
 
+## Helpers
+
+The `xk6-kubernetes` extension offers helpers to facilitate common tasks when setting up a tests. All helper functions work in a implicit namespace to facilitate the development of tests segregated by namespace. The helpers are accessed using the following methods:
+
+|  Method      | Parameters|   Description |
+| -------------| ---| ------ |
+| helpers      | | returns helpers that operate in the default namespace |
+| namespacedHelpers | namespace | returns helpers that operate in the given namespace |
+
+
+The methods above return an object that implements the following helper functions:
+
+|  Method     | Parameters|   Description |
+| ------------ | --------| ------ |
+| createRandomNamespace| prefix         | creates a new namespace with the given prefix and returns its name.  |
```

Comment:
This one feels strange if, on the "helpers/namespacedHelpers" method, I have set a namespace.

Maybe it should be created if, once passed to the helper's method, it doesn't exist (?). 

Also, I think that in this case, the randomness part is too much. 

Anyone should be able to use https://k6.io/docs/javascript-api/jslib/utils/randomstring/ with the same ease 🤔

Reply:
I'm strongly against creating the namespace if it doesn't exist because it will hide the case when the name is mistaken and introduce some hard to debug errors. 

If you consider it is confusing I prefer to remove. However, i found it handy in the e2d tests.

Reply:
I'd concur that having the ns automatically created could be bad; I'd prefer to have the creation of a new namespace be very explicit. I don't care for the "randomness" aspect either; I'd prefer script authors create their own random name to be passed in for creation.